oak_functions/loader/fuzz/fuzz_targets/wasm_invoke.rs

```diff
@@ -107,8 +125,12 @@ impl From<&ArbitraryInstruction> for crate::proto::Instruction {
                 }),
             ),
             ArbitraryInstruction::StorageGetItem { key } => {
+                let key = match key {
+                    LookupKey::FixedKey => FIXED_KEY.to_vec(),
+                    LookupKey::RandomKey { key } => key.clone(),
+                };
```

Comment:
BTW this may also be a method `LookupKey::value()` directly, if you prefer (it seems nicely self contained). This is fine too though.

Reply:
This is nicer. Added a `From` implementation.